tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -58,9 +60,13 @@ def get_config_help(self):
         return config_help
 
     def set_destinations(self):
-        # TODO(anmolbabu):User configuration will not just be destination email
-        # but it would also include user level subscriptions the capability
-        # underneath needs to be enhanced for that.
+        # TODO(gowtham):Get user ids from indexes who enabled
+        # email notification
```

Comment:
@anivargi @anmolbabu here i have to find user id or name  from indexes based on who are all enabled email notification, so what is the path for that indexes and how it looks like (structure).